src/colvarproxy_volmaps.h

```diff
@@ -81,18 +81,22 @@ class colvarproxy_volmaps {
   /// \param atom_end Iterator pointing past the last atom
   /// \param value Pointer to location of total to increment
   /// \param atom_field Array of atomic field values (if NULL, ones are used)
+  /// \param inside Array of flags (1 = inside map, 0 = outside)
   virtual int compute_volmap(int flags,
                              int volmap_id,
                              cvm::atom_iter atom_begin,
                              cvm::atom_iter atom_end,
                              cvm::real *value,
-                             cvm::real *atom_field);
+                             cvm::real *atom_field,
+                             int *inside);
```

Comment:
I will need to update the signature change in https://github.com/Colvars/colvars/pull/788 as well .